Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerEngine.ps1

```diff
@@ -1495,6 +1495,64 @@
         }
     }
 
+    Function Test-DownloadDomainsConfiguration {
```

Comment:
Do not add this function here into the `Invoke-AnalyzerEngine.ps1` file. This function should only be analyzing the object.

Reply:
We can probably keep the function here for now. But we need to update it so it doesn't have the requirement of running any "get" commands. We should already have the data here to be analyzed.

Reply:
Okay, should I create a project file for this new feature under: `DataCollection\ExchangeInformation` ?

Reply:
Done.